app/assets/javascripts/map.js

```diff
@@ -125,25 +207,210 @@ class Map {
         position: results[0].geometry.location
       });
       this.geocodedAddress = results[0].formatted_address;
-      this.onGeocodingResponse(this.geocodedAddress);
+      this.updateAddressOnSummary(this.geocodedAddress);
     } else {
       alert('Geocode was not successful for the following reason: ' + status);
     }
   }
 
+  updateAddressOnSummary(geocodedAddress) {
+    if (geocodedAddress) {
+      this.geocodedAddress = geocodedAddress;
+      const primaryAddressNode = document.getElementById('primaryAddress');
+      const secondaryAddressNode = document.getElementById('secondaryAddress');
+      while (primaryAddressNode.firstChild) { primaryAddressNode.removeChild(primaryAddressNode.firstChild) }
+      while (secondaryAddressNode.firstChild) { secondaryAddressNode.removeChild(secondaryAddressNode.firstChild) }
+      
+      let splitAddress = geocodedAddress.split(',');
+      const div = document.createElement('div');
+      div.innerText = splitAddress.shift();
+      div.classList.add('primary-address');
+      primaryAddressNode.appendChild(div);
+      splitAddress.forEach(element => {
+        const div = document.createElement('div');
+        div.innerText = element;
+        div.classList.add('secondary-address');
+        secondaryAddressNode.appendChild(div);
+      });
+    Dom.showNode(document.getElementById('summary'));
```

Comment:
I would either make `Dom.showNode` take the ID as the argument, or make it so it can take _either_ an ID or a node